### PR TITLE
change: add information about commercial support in the default index…

### PR DESCRIPTION
**Pull request by dndx**
….html.

I hereby granted the copyright of the changes in this pull request
to the authors of this openresty project.

Rendered:
image

### CODE REVIEW

File: html/index.html [hunk 1]
```diff
@@ -16,7 +16,9 @@
 working. Further configuration is required.</p>
 
 <p>For online documentation and support please refer to
-<a href="https://openresty.org/">openresty.org</a>.<br/></p>
+<a href="https://openresty.org/">openresty.org</a>.<br/>
+Commercial support is available at
```
Comment (agentzh) [inline html/index.html]: Better introduce a new paragraph here?

Comment (dndx) [reply html/index.html]: There is no space in the default one shipped with NGINX. But if you want I can certainly add it.

Comment (agentzh) [reply html/index.html]: @dndx Not a big deal. We can merge the current form for now :)

**Comment by agentzh** [comment]
Merged. Thanks!

**Pull request closed without merging**